dog.md

```diff
@@ -0,0 +1,204 @@
+```k
+requires "kmcd-driver.md"
+requires "clip.md"
+requires "vat.md"
+requires "vow.md"
+
+module DOG
+    imports KMCD-DRIVER
+    imports PRE-CLIP
+    imports VAT
+    imports VOW
+```
+
+Dog Configuration
+-----------------
+
+```k
+    configuration
+      <dog-state>
+         <dog-vat>   0:Address </dog-vat>
+         <dog-ilks>  .Map      </dog-ilks>   // mapping (bytes32 => Ilk) String |-> DogIlk
+         <dog-vow>   0:Address </dog-vow>
+         <dog-live>  true      </dog-live>
+         <dog-hole>  rad(1)    </dog-hole>
+         <dog-dirt>  rad(1)    </dog-dirt>
```

Comment:
On initial values--to match the contract behavior, these should start at zero. It doesn't matter much for `dog-hole` which is a governance parameter, but it actually is important for `dog-dirt` which needs to start at zero.